collections/_members/yash-pandey.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: person
+name: Yash Pandey
+position: Contributor
+image_path: /assets/images/community/members/yash-pandey.jpg
```

Comment:
Can you keep this image in the same folder?

Reply:
@Tanuj22  Same folder as in??

Reply:
Keep the image in the same folder as index.mdx file

Reply:
Done.

Reply:
> @Tanuj22 Same folder as in??

Sorry just seeing this. @96RadhikaJadhav you need to update your master according to the latest fork. You have made a PR from the old site so this is creating a folder `collections/_members` whereas it should be in `src/collections/members/`

Reply:
@Tanuj22 @Jashpatel1 I have updated master but somehow, the changes were not being reflected to my branch. So I created another branch and pushed the changes. Can you please close this PR and check #1607 .